### PR TITLE
add docs for resetting passphrases

### LINKED ISSUE #3387: Provide documentation on how to rotate journalist's passphrase
# Feature request

Provide documentation on how an admin can rotate journalist's passphrase in the SecureDrop app

## Description

It is currently possible to rotate a journalist's user passphrase in the SecureDrop web interface, but it doesn't appear to be documented anywhere.

## User Stories

As a user, I'd like to be able to get my passphrase reset, and ask my organization's internal IT for help. As an IT administrator of a SecureDrop instance, I'd like to help users reset/rotate their passphrases.

### PR DESCRIPTION
**Pull request by b-meson**
## Status

Ready for review

## Description of Changes

Fixes #3387.

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

File: docs/admin.rst [hunk 1]
```diff
@@ -183,18 +180,23 @@ SecureDrop. Make sure the journalist remembers their username and
...
+While we publish some :ref:`passphrase best practices <passphrase_best_practices>`,
+your journalists may occasionally lock themselves out of their accounts. This
+can happen if, for example, they lose their two-factor device or if they
+forget the passphrase  to their password manager. When this happens, you
+can reset  their account by logging in as an administrator to the
+*Journalist Interface* and selecting *Admin* at the top right. Find their
+username and select **Edit**. Next, you can either rotate their passphrase or
+reset their two-factor authentication. You should make sure the user's
+passphrase is saved in a password manager *before* selecting "Reset Password".
+There are two options for resetting two-factor authentication depending on it
```
Comment (edenemmanuel) [inline docs/admin.rst]: "There are two options for resetting two-factor authentication depending on it was configured during on-boarding."

I'm not sure what you mean here, I must admit.

File: docs/admin.rst [hunk 2]
```diff
@@ -394,3 +394,14 @@ by monitoring OSSEC alerts. (Please note that while an OSSEC alert can notify yo
 occurrence of an update to the server, it may not reveal the content of the change.) Another
 management option would be SSHing into the server and manually inspecting the configuration to
 identify any discrepancies.
+
+.. |Reset Passphrase| image:: images/manual/screenshots/password_reset.png
```
[the remark this answers is not in the document]
Comment (b-meson) [reply docs/admin.rst]: Thanks @redshiftzero. I should note that this description is counter intuitive: 

> Note: all images in the `images/manual/screenshots/*` directory are automatically generated using Selenium. 

In fact, I put them there based on the name of the file path. I should note that when I first got started on the SD project, you mentioned there was an image path that was auto generated using Selenium. If I couldn't recall remember it, then its probably indicative of being unapproachable to most contributors.

[the remark this answers is not in the document]
Comment (redshiftzero) [reply docs/admin.rst]: so it turns out there is actually an existing screenshot in our page layout tests that should do the job here, it's [`journalist-edit_account_admin.png`](https://github.com/freedomofpress/securedrop/blob/develop/securedrop/tests/pages-layout/test_journalist.py#L275) - so if you run the auto-screenshot-generation as described in the documentation I link to above and simply add that file in the docs here, you should be good to go! Also I can add a commit that does this if you like

**Comment by b-meson** [comment]
I'll take a look at it this weekend @redshiftzero and make the changes.

**Comment by redshiftzero** [comment]
Thanks @b-meson, this will be a nice improvement to the documentation!

**Comment by b-meson** [comment]
Due to the new SecureDrop workflow I have added @pierwill on my fork to collaborate on this PR.

**Comment by b-meson** [comment]
Closing in favor of #3638. Why? I can no longer push to my old branch on the SD repository so I am making a PR from my fork.

**Pull request closed without merging**